### PR TITLE
SAR-10340 validate input value for turnover, zero rated turnover, NIP

### PR DESCRIPTION
**Pull request by kaizar12**
SAR-10340 validate input value for turnover, zero rated turnover, NIP

**New feature**

## Checklist

* [x] I've included appropriate tests with any code I've added
* [ ] I've executed the acceptance test pack locally to ensure there are no regressions
* [x] I've added my code using logical, atomic commits, squashing as appropriate - including the Jira issue number in the commit message
* [ ] I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -679,7 +684,7 @@ cya.registrationDetails.companyBankAccount.reason               = Reason for no
 cya.registrationDetails.joinFrs                                 = Do you want to join the Flat Rate Scheme?
 cya.registrationDetails.costsInclusive                          = Will the business spend more than £250 over the next 3 months on ’relevant goods’?
 cya.registrationDetails.estimateTotalSales                      = Total sales, including VAT, for the next 3 months
-cya.registrationDetails.costsLimited                            = Will the business spend more than £{0}, including VAT, on relevant goods over the next 3 months?
+cya.registrationDetails.costsLimited                            = Will the business spend more than {0}, including VAT, on relevant goods over the next 3 months?
```
Comment (saojones) [inline conf/messages:687]: What is this change for?

File: conf/messages [hunk 2]
```diff
@@ -1289,20 +1296,26 @@ pages.CoverSheet.vat1TRLink                        = VAT1TR form
...
 # NORTHERN IRELAND PROTOCOL
 
 nip.sellOrMove.heading                                   = Does the business expect to do any of the following in the next 12 months?
 nip.sellOrMove.para                                      = Tell us if the business will do any of the following:
 nip.sellOrMove.bullet1                                   = sell goods located in Northern Ireland to any country
 nip.sellOrMove.bullet2                                   = sell or move goods from Northern Ireland to an EU country
-nip.sellOrMove.hint                                      = What is the value of these goods?
-nip.error.missing                                        = Select yes if the business will do any of the activities in the next 12 months
-validation.sellOrMoveNip.missing                         = Enter the value of these goods
+nip.sellOrMove.label                                     = What is the value of these goods?
```
Comment (saojones) [inline conf/messages:1312]: What is the *estimated value of these goods?

File: conf/messages [hunk 3]
```diff
@@ -963,7 +970,7 @@ pages.frs.registerForWithSector.subheading1               = Do you want the busi
 validation.frs.registerForWithSector.missing              = Select yes if you want the business to join the Flat Rate Scheme
 
 ## Do you spend less than 2% of your turn over Page
-pages.frs.costsLimited.heading                          = Will the business spend more than £{0}, including VAT, on relevant goods over the next 3 months?
+pages.frs.costsLimited.heading                          = Will the business spend more than {0}, including VAT, on relevant goods over the next 3 months?
 validation.frs.costsLimited.missing                     = Select yes if the business will spend more than £{0}, including VAT, on relevant goods over the next 3 months
```
Comment (saojones) [inline conf/messages:974]: £ not removed here

File: conf/messages (hunk 2 above)
Comment (kaizar12) [reply conf/messages:1312]: We are changing hint and error messages under this ticket, that one is a content change so not changing it here.

**Pull request merged**